packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.Props.ts

```diff
@@ -66,4 +67,112 @@ export interface ICheckboxProps extends React.HTMLAttributes<HTMLElement | HTMLI
    * @default 'start'
    */
   boxSide?: 'start' | 'end';
+
+  /**
+   * Theme provided by HOC.
+   */
+  theme?: ITheme;
```

Comment:
I do not fully understand how the theme should work. I based this part of the change on the Toggle component which already has moved onto the Glamor based styling.